src/release_notes/model/custom_chapters.py

```diff
@@ -0,0 +1,40 @@
+import json
+import logging
+
+from release_notes.model.base_chapters import BaseChapters
+from release_notes.model.chapter import Chapter
+from release_notes.model.record import Record
+
+
+class CustomChapters(BaseChapters):
+    """
+    A class used to represent the custom chapters in the release notes.
+    """
+
+    def populate(self, records: dict[int, Record]):
+        """
+        Populates the custom chapters with records.
+
+        :param records: A dictionary of records where the key is an integer and the value is a Record object.
+        """
+        for nr in records.keys():                               # iterate all records
```

Comment:
`.keys` is not needed I believe - that's the default one

Reply:
You are right. Removed in commit b5dba340a8703ee5c6c955537da6d22530240aa9.